### ISSUE TITLE
Slow Uninstallation Progress Compared with Squirrel.Windows

### ISSUE THREAD

**Issue by yuliaria** [clowd/Clowd.Squirrel#109]
Hi All, 

I am upgrading my project from squirrel.windows to clowd.squirrel because I need to use .net 6. 

When I uninstall my program, it takes around 12-14 seconds. 
squirrel.windows took only around 2-3 seconds. 

My program got a lot of executable files (both C# and C++ code), and clowd.squirrel creates stub files for all these executable files. 
Squirrel.windows also created stub files, but it was only for C# codes. 

Most likely this is the cause of the slow uninstall.

Is there any option to turn off the stub executable files creation?

Thank you.

**Comment by caesay** [comment]
Squirrel.Windows only creates stub's for GUI subsystem PE files. Clowd.Squirrel creates stub files for GUI and Console subsystem files. This feature allows people to create shortcuts for console application and it is not configurable. This also has nothing to do with uninstalling.

There are two things that happen during uninstall:
 - Your "SquirrelAware" applications will all be run, one by one, with the `--squirrel-uninstall` flag. Squirrel will wait for your application to exit before continuing.
 - The install directory and registry entries will be deleted.

I do not know how you are measuring uninstall (since this is completely invisible to the user), however if there is some delay it is most likely your application not exiting quickly, or some issue with your disk speed.

**Comment by yuliaria** [comment]
Hi Caesay,

Thank you for your kind response. 

I measure time from the moment I pressed uninstall until the physical files are removed.

I actually have a lot of exe files written in C++, I could say more than 500. 
While my GUI subsystem PE files only 5. 

So, the installation and uninstallation are somewhat slower now.

I actually wrote my own code to create shortcuts, and not use the stub executables.
That's why I was wondering if I could disable the stub file creation or not.

But, it is clear to me now that it is not configurable. So, I will just need to accept the new behavior. :)

Once again thank you very much for answering my question.
